packages/api/src/utils/client/httpClient.ts

```diff
@@ -181,6 +182,18 @@ export class HttpClient implements IHttpClient {
   private async requestWithBodyWithRetries<T>(
     opts: FetchOpts,
     getBody: (res: Response) => Promise<T>
+  ): Promise<{status: HttpStatusCode; body: T}> {
+    return retry(
+      async (_attempt) => {
+        return this.requestWithBodyWithFallbacks<T>(opts, getBody);
```

Comment:
I am not so sure this is the behavior we want if fallback urls are configured + retryAttempts > 1. Instead of retrying the whole fallback flow (which has internal state) I'd rather see that we retry individual requests to each fallback url.

Reply:
i don't think individual requests should be retried because its crucial for requests to go to the fallbacks as soon as possible. 
Also keeping retrying logic simple is essential exactly because of internal state of fallbacks, i.e. just simply retry the whole thing.

Reply:
> i don't think individual requests should be retried because its crucial for requests to go to the fallbacks as soon as possible.

That's also a valid point, either way it does not feel like retry and fallback work that well together or rather retries do not seem that useful in combination with fallback nodes.

Since we only use this with builder which has no fallback this looks good to me.